packages/ui/src/components/Select/select.tsx

```diff
@@ -40,7 +40,7 @@ const Select: ISelect = (props) => {
         [`${SELECT_CLASS_PREFIX}-${type}`]: type,
     })
 
-    const handleChange = useCallback((value, option) => {
+    const handleChange = useCallback((value: any, option: any) => {
```

Comment:
`string | number | null` instead of `any`, as described within `OptionType`
Or create a custom type for an option value `type OptionValueType = string | number | null`

Reply:
Option has children here...And analytics receives only strings...So idk about this type implementation. Strict typing here needed for dev-portal to build (yeah, magic :) )